assignment7/src/style.css

```
@@ -0,0 +1,311 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300;600;800&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+:root {
+  --body-font: "Nunito Sans", sans-serif;
+  --light-mode-text: hsl(200, 15%, 8%);
+  --light-mode-background: hsl(0, 0%, 98%);
+  --light-mode-input: hsl(0, 0%, 52%);
+  --white: hsl(0, 0%, 100%);
+  --dark-mode-bg: hsl(207, 26%, 17%);
+  --dark-mode-elements: hsl(209, 23%, 22%);
+  --purple: rgb(143, 43, 236);
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  background-color: var(--light-mode-background);
+  color: var(--light-mode-text);
+  font-family: var(--body-font);
+  font-size: 1.8rem;
+  position: relative;
+}
+.app {
+  max-width: 1440px;
+  margin: auto;
+}
+img {
+  display: block;
+}
```

```suggestion
}

```
Add spaces between 2 style defs